forkrun.bash

```diff
@@ -1681,3 +1738,114 @@
 }
 
 _forkrun_lseek_setup
+
+
+export -fp _forkrun_get_load &>/dev/null && export -nf _forkrun_get_load
+
+_forkrun_get_load() (
```

Comment:
The efficiency of _forkrun_get_load is less critical than you might think. This doesnt even come into play until some initial worker coprocs are forked, and (since this is being called from a coproc itself) the only think that is "slowed down" by waiting for this is how rapidly additional coprocs are forked. Additionally, dynamically determining coproc count is more about reducing CPU usage than it is about reducing the runtime, so slowing down how rapidly now worker coprocs are forked is actually good for efficiency (reducing CPU usage) and helps ensure not too many are forked (since once they are forked they run until the end)